plugins/examples/webrtcsink-stats-server.rs

```diff
@@ -133,6 +133,24 @@ async fn run(args: Args) -> Result<(), Error> {
         .by_name("ws")
         .unwrap();
 
+    ws.connect("encoder-setup", false, |values| {
```

Comment:
This `false` here makes it connect *before* the class handler btw, a `true` would be after.